ignite/src/main/java/org/hibernate/ogm/datastore/ignite/impl/IgniteDatastoreProvider.java

```diff
@@ -328,6 +328,7 @@ public void clearCache(String cacheName) {
 	 */
 	private static class ComputeForLocalQueries<T> implements IgniteCallable<List<T>> {
 
+		private static final long serialVersionUID = 1L;
```

Comment:
We don't add the serial version id to classes unless there are good reason to have it.

Reply:
ok. I will remove it. Eclipse asks

Reply:
I know but it is a warning. Let it ask :)

Reply:
fixed